app/grants/templates/grants/shared/landing_grants.html

```diff
@@ -2,38 +2,33 @@
 
     <div class="container-fluid grants-container">
       <div class="container">
-        {% if grants %}
-          <div class="infinite-container row">
-            {% for grant in grants %}
-              <div class="col-12 col-md-6 col-xl-4 mb-4 infinite-item">
-                {% include 'grants/card/index.html' %}
-              </div>
-            {% endfor %}
+        <div class="infinite-container row" v-if="grants">
```

Comment:
I think scroll method is not being call here so probably pagination will not work you need to use the directive `scroll.passive` (check notifications.html)